src/app/services/sms/__tests__/sms.factory.spec.ts

```diff
@@ -24,15 +23,6 @@ const MOCKED_TWILIO = {
   mocked: 'this is mocked',
 } as unknown as Twilio.Twilio
 
-const MOCK_BOUNCE_SMS_PARAMS: BounceNotificationSmsParams = {
```

Comment:
why do we need to remove these?

Reply:
No longer used. For admins, we're switch them directly over to Postman.